extensions/oidc/runtime/src/main/java/io/quarkus/oidc/TenantResolverInterceptor.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.oidc;
+
+import jakarta.inject.Inject;
+import jakarta.interceptor.AroundInvoke;
+import jakarta.interceptor.InvocationContext;
+
+import io.quarkus.oidc.runtime.OidcUtils;
+import io.vertx.ext.web.RoutingContext;
+
+public abstract class TenantResolverInterceptor {
+    @Inject
+    RoutingContext routingContext;
+
+    @AroundInvoke
+    public Object setTenant(InvocationContext context) throws Exception {
```

Comment:
Just curious, would `public Object setTenant(InvocationContext context, RoutingContext routingContext) throws Exception` work here?

Reply:
@gastaldi I see what I mean, let me try

Reply:
@gastaldi I could not make it work unfortunately

Reply:
That's fine, probably the spec doesn't allow it